packages/babel-plugin-transform-remove-console/test/fixtures/remove-console/statement-no-block/actual.js

```diff
@@ -0,0 +1,3 @@
+if (blah) console.log(blah);
+for (;;) console.log(blah);
+while (blah) console.log(blah);
```

Comment:
It looks done in [this step of removal-hooks](https://github.com/babel/babel/blob/master/packages/babel-traverse/src/path/lib/removal-hooks.js#L9), it will transform to `var a = (blah) => void 0;`, should we add the case to expression fixture?